src/docs/common/snippets/common-graalpy-graalvm.adoc

```diff
@@ -0,0 +1,71 @@
+:exclude-for-languages:groovy
+
+== Python Runtime Compilation
+
+With GraalVM JDK, the Python code executed using the GraalPy engine will be just-in-time compiled.
+With other JDKs, the Python code will be interpreted, which negatively impacts its performance.
+
+The easiest way to install https://www.graalvm.org[GraalVM] on Linux or Mac is to use https://sdkman.io/[SDKMan.io].
+
+[source, bash]
+.Java 23
+----
+sdk install java 23-graal
```

Comment:
23-graal won't work with Gradle afaik

Reply:
I assume this guide will be only generated for Maven?

Reply:
Yes, correct